### ISSUE TITLE
Ruleset of newly created processes

### ISSUE THREAD

**Issue by IkramMaalej** [kitodo/kitodo-production#4941]
When creating a new process from a template, its ruleset will automatically match the selected template's ruleset. Similarly, if the creation is done from the 'process list' page using the "+" or "Calendar" buttons, then the ruleset of the newly created process is the same as the selected process' template and not the same as the ruleset of the selected process.

In addition, the user can edit the russets process from the 'Edit Process' page, therefore a process and its template can have different rulesets. As a result, creating a child process will end up having a child process with a different ruleset than its parent. 
This behaviour could also occur when creating processes from a calendar

Question: Is this an intended behaviour, or we should consider it as a Bug?

**Comment by matthias-ronge** [comment]
Parent and child processes should always have the same ruleset, so this is a bug.

**Comment by BartChris** [comment]
So what exactly has to be fixed here? Should we completely remove the possibility to edit the ruleset via the process edit page?

**Comment by IkramMaalej** [comment]
> So what exactly has to be fixed here? Should we completely remove the possibility to edit the ruleset via the process edit page?

Yes, I think , since a process is created, it shouldn't be possible to change its ruleset.

**Comment by solth** [comment]
There is still the option to change the ruleset of existing processes using the corresonding [KitodoScript](https://github.com/kitodo/kitodo-production/wiki/KitodoScript#setruleset), though. So before deactivating the option to set the ruleset of a process, perhaps we should first probe how important this is during daily use. 
Looking at you, @kitodo/kitodo-community-board !

**Comment by andre-hohmann** [comment]
The @kitodo/kitodo-community-board recommends maintaining the functionality to change the ruleset of a process. 
It can be necessary to adjust existing processes after rulesets have been merged or separated.

> Parent and child processes should always have the same ruleset, so this is a bug.

We are unsure why this restriction should apply. 
For example if there is multivolume with an text volume and a audio volume, where text and audio objects use different rulesets. 

However, displaying a warning when the rulesets of subordinate and superordinate processes differ could be a helpful compromise.
